utils/auth_jwt.ts

```diff
@@ -0,0 +1,70 @@
+import * as t from "io-ts";
+
+import * as E from "fp-ts/Either";
+import * as TE from "fp-ts/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+
+import { Second } from "@pagopa/ts-commons/lib/units";
+import { AssertionRef } from "../generated/definitions/internal/AssertionRef";
+import { OperationId } from "../generated/definitions/internal/OperationId";
+
+import { getGenerateJWT, getValidateJWT } from "./jwt_with_key_rotation";
+
+import { JWTConfig } from "./config";
+
+/**
+ * Type Definitions
+ */
+
+export type AuthJWT = t.TypeOf<typeof AuthJWT>;
+export const AuthJWT = t.interface({
+  assertionRef: AssertionRef,
+  operationId: OperationId
+});
+
+export type DecodedAuthJWT = AuthJWT;
```

Comment:
Do we need this?

Reply:
Actually, not anymore. Removed in [[#IOCIT-301] remove DecodedAuthJWT + minor fix](https://github.com/pagopa/io-functions-lollipop/pull/14/commits/915afe8d302a5022519a7819395f187c71a9c485)